src/components/user/UserMenu.js

```diff
@@ -25,7 +27,7 @@ const UserMenu = ({ children }) => {
         >
           <Menu.Item as={"div"}>
             <Icon name="user" />
-            {user.email}
+            {data}
```

Comment:
Can we do something better that `data` in terms of naming?

Reply:
hmm good question :D the problem is that data is props coming from the hook and I'm not sure If I can change it, gonna try

Reply:
you can do this:
```javascript
const { data: yourSuperName } = useQuery(["user-nickname", user.uid], (key, id) => getNickName(id));
```

Reply:
done